### PR TITLE
Feature/light

### PR DESCRIPTION
**Pull request by SiggyF**
Add some lights

### CODE REVIEW

File: sdc-static/src/lib/contours.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ function addObjectLayer(map, id, url, color, metadata) {
   // todo, use metadata properly
 
   // parameters to ensure the model is georeferenced correctly on the map
-  var multiplyZ = 0.000005
+  var multiplyZ = 0.000001
```
Comment (CindyvdVries) [inline sdc-static/src/lib/contours.js:9]: Is this the max multiplyer? Please put thta in the comment, so we know why this number

File: sdc-static/src/lib/contours.js [hunk 2]
```diff
@@ -51,14 +51,31 @@ function addObjectLayer(map, id, url, color, metadata) {
 
 
       // TODO: replace by more appealing light (this one doesn't  cast shadows)
-      const sky = 0xffffff
-      const ground = 0xB97A20
-      var light = new THREE.HemisphereLight(sky, ground, 1);
-      this.scene.add(light);
+      // const sky = 0xffffff
```
Comment (CindyvdVries) [inline sdc-static/src/lib/contours.js:54]: remove comments?

File: sdc-static/src/lib/contours.js [hunk 3]
```diff
@@ -89,19 +106,51 @@ function addObjectLayer(map, id, url, color, metadata) {
             emissive: 0x333333,
             opacity: 0.5
           });
-          var mesh = new THREE.Mesh(geometry, material);
-          mesh.doubleSided = false;
+          // var meshMaterial = new THREE.MeshStandardMaterial( { color: color } );
```
Comment (CindyvdVries) [inline sdc-static/src/lib/contours.js:109]: remove lines that are not used

File: sdc-static/src/lib/contours.js [hunk 4]
```diff
@@ -89,19 +106,51 @@ function addObjectLayer(map, id, url, color, metadata) {
...
+          // mesh.doubleSided = false;
 
           // mesh.rotation.z = Math.PI;
 
-          this.scene.add(mesh);
-          mesh.scale.z = multiplyZ;
-          // mesh.translateZ(0.005)
           mesh.castShadow = true;
           mesh.receiveShadow = true;
+          mesh.scale.z = multiplyZ;
```
Comment (CindyvdVries) [inline sdc-static/src/lib/contours.js:118]: ; ?

**Review by CindyvdVries — APPROVED** [review]
Looks good and nice results! 
Please tidy up a bit

File: sdc-static/src/lib/contours.js (hunk 1 above)
Comment (SiggyF) [reply sdc-static/src/lib/contours.js]: added comments and proper calculation

File: sdc-static/src/lib/contours.js (hunk 2 above)
Comment (SiggyF) [reply sdc-static/src/lib/contours.js]: cleaned up

File: sdc-static/src/lib/contours.js (hunk 4 above)
Comment (SiggyF) [reply sdc-static/src/lib/contours.js:112]: explained

**Pull request merged**